### DISCUSSION TITLE
Improved Landing Page ?

### DISCUSSION THREAD

**Discussion by Nisarg38** [benawad/dogehouse#2406] [Ideas]
I had few ideas to improve the landing page for DogeHouse, Should I create a mock-up?

**Comment by epic-developer** [comment]
Yeah, it would be nice if you sent a mockup. I think it could just list some features and display those features in video clips. I'm thinking it could look similar to the landing page for [Repl.it](https://replit.com/). Just list the main features of Dogehouse in some simple, short sentences (obviously focus on the features that make it better than other services for certain use cases).
